src/axolotl/utils/chat_templates.py

```diff
@@ -2,9 +2,36 @@
 This module provides functionality for selecting chat templates based on user choices.
 These templates are used for formatting messages in a conversation.
 """
+import logging
+from typing import TYPE_CHECKING, Any, Dict, Optional
 
+if TYPE_CHECKING:
+    from transformers import PreTrainedTokenizerBase
 
-def chat_templates(user_choice: str):
+LOG = logging.getLogger("axolotl.utils.chat_templates")
+
+_JINJA_TEMPALTE_CHOICE = "jinja"
+_DEFAULT_TEMPLATE_CHOICE = "tokenizer_default"
+_DEFAULT_FALLBACK_CHATML_TEMPLATE_CHOICE_PREFIX = "tokenizer_default_fallback_"
```

Comment:
Could you tell me what's the difference between the two? How would one explicitly set the config to use the tokenizer?

```yaml
type: chat_template
chat_template: tokenizer_default
```

Is this correct? I think if this part is clear, we can close https://github.com/axolotl-ai-cloud/axolotl/pull/1855/files .

Reply:
Yes, this is correct, updated docs as per your review comment